x/concentrated-liquidity/types/codec.go

```diff
@@ -15,10 +15,6 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 }
 
 func RegisterInterfaces(registry cdctypes.InterfaceRegistry) {
-	registry.RegisterInterface(
-		"osmosis.concentratedliquidity.v1beta1.PoolI",
-		(*ConcentratedPoolExtension)(nil),
-	)
```

Comment:
for my learning, why do we no longer need to register this in CL? Is it because we never utilize PooI and instead utilize ConcentratedLiquidityExtension? Does it matter that ConcentratedLiquidityExtension imports PoolI? I might be a step ahead though and this might need to get reintroduced again later, just want to see if my understanding was off.

Reply:
This is because we don't marshal this as `Any` to write to the store. Instead, we directly type cast and write to store which is cleaner than dealing with "Any" marshaling complexity